charts/security-apps/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: security-apps
 description: Argo CD app-of-apps config for security applications
 type: application
-version: 0.89.0
```

Comment:
Imo this should be a minor release, not a patch release

Reply:
Else if someone is using 0.89.* their deployment may fail without user interaction

Reply:
Are there still folks that do have `x.z.*`, i thought we deprecated that ages ago

Reply:
It's a public chart, so we may not be the only consumers. It's also a breaking change in a patch version. A minor version is more of a signal that "hey, we may need to change something" for consumers so I believe it's a better fit here.

Reply:
Thanks for the review, i finally got to updating the PR to bump the minor and not the patch version.

I totally agree with you in principal, the main reason i asked was because we used to employ the `x.y.*` all over the place and we started retiring it's use a few years back.